---
date: 2018-11-21
slug: 2018-gutschow-combinatorial
image: 2018-gutschow-combinatorial.png
title: "Combinatorial processing of bacterial and host-derived innate immune stimuli at the single-cell level"
authors: Miriam V. Gutschow, John C. Mason, Keara M. Lane, Inbal Maayan, Jacob J. Hughey, Bryce T. Bajar, Debha N. Amatya, Sean D. Valle, and Markus W. Covert
preprint_journal: 
preprint_doi: 
short_ref: Gutschow et al., Mol Biol Cell 2018
peer_reviewed: true
journal: Molecular Biology of the Cell
year: 2018
doi: 10.1091/mbc.E18-07-0423
pdf: 2018-gutschow-combinatorial.pdf
supp: 
pmid: 30462580
reviews: 
results: 
software: 
member_ids: hugheyjj
software_ids: 
---

# Abstract

During the course of a bacterial infection, cells are exposed simultaneously to a range of bacterial and host factors, which converge on the central transcription factor nuclear factor (NF)-κB. How do single cells integrate and process these converging stimuli? Here, we tackle the question of how cells process combinatorial signals by making quantitative single-cell measurements of the NF-κB response to combinations of bacterial lipopolysaccharide (LPS) and the stress cytokine Tumor Necrosis Factor (TNF). We found that cells encode the presence of both stimuli via the dynamics of NF-κB nuclear translocation in individual cells, suggesting the integration of NF-κB activity for these stimuli occurs at the molecular and pathway level. However, the gene expression and cytokine secretion response to combinatorial stimuli were more complex, suggesting that other factors in addition to NF-κB contribute to signal integration at downstream layers of the response. Taken together, our results support the theory that during innate immune threat assessment, a pathogen recognized as both foreign and harmful will recruit an enhanced immune response. Our work highlights the remarkable capacity of individual cells to process multiple input signals and suggests that a deeper understanding of signal integration mechanisms will facilitate efforts to control dysregulated immune responses.
